.rat-excludes

```diff
@@ -66,3 +66,8 @@ google-big-query.svg
 google-sheets.svg
 postgresql.svg
 snowflake.svg
+
+# docs-related
+docs/docs/.htaccess*
+erd.plantuml
+erd.svg
```

Comment:
We could probably add the license boilerplate to the SVG as part of the action, if that makes sense to do so.